pages/infrastructure_providers-validators/running_full_node.md

```diff
@@ -1,73 +1,60 @@
-
-### Starting a Full Node
-
-Find the seed node's ID and the IP address from [Resources](../infrastructure_providers-network/resources.mdx#seed-nodes). Then, run the following command to start a non-validating full node.
-
-For example,
-```bash
-dydxprotocold start --p2p.seeds="..." --bridge-daemon-eth-rpc-endpoint="<eth rpc endpoint>" --non-validating-full-node=true
-```
-
-💡**Note**: if you want to disable gRPC on your full node, it is important to start the node with the
-`--non-validating-full-node=true` flag. Otherwise, the application will require that gRPC be enabled.
-
-
-## Staying up-to-date with the Latest State
-
-### Connecting to Healthy Peers
-
-In order for the full node to have the latest state, it needs to connect to peers which have the latest state Use the following links to get a list of live peers which have the latest state
-
-https://services.lavenderfive.com/mainnet/dydx#live-peers
-
-https://polkachu.com/live_peers/dydx
-
-Update `persistent_peers`  in the config.toml file to include a randomly selected list of 5 peers from the list of live peers obtained using the links above
-
-### Snapshots
-
-Snapshots contain a compressed copy of the chain data which allow the full node to bootstrap to a recent state in the blockchain. A list of snapshot services can be found [here](../infrastructure_providers-network/resources.mdx#snapshot-service).
-
-### State sync (Alternative to snapshots)
-
-State Sync enables a new node to join the network by obtaining a snapshot of the application state from a state sync node at a recent height. This eliminates the need to fetch and replay all historical blocks. A list of state sync services with instructions are listed below
-
-https://polkachu.com/state_sync/dydx
-
-https://services.lavenderfive.com/mainnet/dydx/statesync
-
-https://autostake.com/networks/dydx/#state-sync
-
-### Address Book
+## Save an Address Book
+Download the latest `addrbook.json` file, which stores configuration details that help your node efficiently connect to peers in its network. Download an up-to-date address book file for your deployment from an [Address Book](../infrastructure_providers-network/resources#address-book-providers) provider.
 
-The **`addrbook.json`** file is used to store configuration details that help a node connect to other peers in the network more efficiently. This can be obtained from the one of the below services and needs to be stored in the `config` folder.
+Save the `addrbook.json` file in your `/.dydxprotocol/config` directory.
 
-https://polkachu.com/addrbooks/dydx
+## Back Up Your Node
+> If you followed the procedure on the previous page, [Set Up a Full Node](../infrastructure_providers-validators/how_to_set_up_full_node.md), you already have a snapshot installed.
 
-https://services.lavenderfive.com/mainnet/dydx#latest-addrbook
+Your full node needs a backup plan to replay the history of the network in case it falls out of sync. You can back up your node in one of two ways.
```

Comment:
Suggestion:

"Your full node can fall out of sync with the rest of the network due to various reasons, including a bad software upgrade, unexpected node crashes or human operational error. In such cases, you need a way to restore your full node so it can catch back up with the network's latest state. You can restore your node in one of two ways:"